src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <div className="App">
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
+
+      {person.age && <p className="Person__age">I am {person.age}</p>}
+
+      {!person.isMarried ? (
+        <p className="Person__partner">I am not married</p>
+      ) : (
+        <p className="Person__partner">
+          {person.partnerName} is my {person.sex === 'm' ? 'wife' : 'husband'}
```

Comment:
Nested ternary looks complicated, let's create a separate variable partner and use it here:

`const partner = person.sex === 'm' ? 'wife' : 'husband'`